src/nestjs-back/src/users/users.controller.ts

```diff
@@ -14,12 +14,27 @@ export class UsersController {
     findAll(@Query() paginationQuery: PaginationQueryDto) {
         return this.usersService.findAll(paginationQuery);
     }
-    
+
     @Get(':id')
     findOne(@Param('id') id: string) {
         return this.usersService.findOne(id);
     }
-    
+
+    @Get('/:id/ownedChannels')
+    getOwnedChannels(@Param('id') id: string) {
```

Comment:
here specialization makes sense because we are getting these specifics items through the route. This is good.